bin/sozo/src/commands/build.rs

```diff
@@ -18,11 +20,14 @@ pub struct BuildArgs {
     #[arg(long)]
     #[arg(help = "Output directory.", default_value = "bindings")]
     pub bindings_output: String,
+
+    #[command(flatten)]
+    pub stats: Stats,
```

Comment:
i think we can directly take a boolean here to keep it simple, wdyt?

Reply:
since we will only use this flag in `build` subcommand i dont think that abstraction is necessary

Reply:
Yes absolutely!